salt/cert/init.sls

```diff
@@ -7,3 +7,22 @@ include:
          pillar['ssl']['crt_file'],
          pillar['ssl']['key_file'],
          o = pillar['certificate_information']['subject_properties']['O']) }}
+
+#######################################
+# additional system wide certificates #
+#######################################
+
+# Install additional certificates that were setup in Velum by the user as
+# system-wide certificates
+
+{% set certs = salt.caasp_pillar.get('system_certificates', []) %}
+{% for cert_tuple in certs %}
+  {% set name, cert = cert_tuple %}
+
+/etc/pki/trust/anchors/{{ name }}.crt:
```

Comment:
Don't we need to issue an `update-ca-certificates` call whenever one of these certs changes?

Reply:
That's true - I added an `onchanges` run, but that would run after every certificate.

If the user adds a many certificates that's really wasteful though, but I am not sure how to only run it after the loop if changes occured (like in `ansible` I would use register, but I didn't find an equivalent for salt).

Reply:
Does the `update-ca-certificates` really work on CaaSP? I remember we had some problems in the past with this command beacuse of the _read-only_ filesystrem...

Reply:
Yes it does nowadays

Reply:
@inercia I just ran it on a openstack deployment and it worked, so seems it does.